css/stylesdesk.css

```diff
@@ -221,125 +178,136 @@
   }
 
   .pr-description {
-    position: relative;
-    width: auto;
-    padding: 0;
     margin: 2% 0;
-    font-weight: 400;
     font-size: 14px;
     line-height: 24px;
-    color: #344563;
-    order: 2;
   }
 
   ul.pr-tech-tags {
-    position: relative;
-    width: auto;
-    height: auto;
     margin: 2% 0;
+  }
+
+  .pr-tag {
+    padding: 4px 10px;
+  }
+
+  .pr-call-to-action-box {
+    margin: 2% 0;
+  }
+
+  .pr-call-to-action-button:link,
+  .about-resume-action-button:link {
+    height: 40px;
+    font-size: 16px;
+    padding: 8px;
+  }
+
+  .modal-popup {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100vh;
+    margin: 0 auto;
     padding: 0;
+    background: rgba(194, 200, 209, 0.85);
+    z-index: 99;
+  }
+
+  .popup-card {
+    height: 85vh;
+    margin: 3rem 4rem auto 4rem;
+    padding: 0.8rem 0.6rem 0.6rem;
+    gap: 0.6rem;
+  }
+
+  .popup-title {
+    width: 100%;
+    height: 2rem;
     display: flex;
     flex-direction: row;
-    justify-content: start;
+    justify-content: space-between;
     align-items: center;
-    gap: 8px;
-    order: 3;
+    order: 0;
   }
 
-  .pr-tag {
-    position: relative;
-    width: auto;
-    height: auto;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    letter-spacing: 0.03em;
-    color: #0400be;
-    background: #ebebff;
-    border-radius: 8px;
-    border: none;
-    padding: 4px 8px;
+  .popup-pr-title {
+    font-size: 32px;
   }
 
-  .pr-call-to-action-box,
-  .about-resume-action-box {
-    position: relative;
-    margin: 0;
-    padding: 4px 0 0 0;
-    width: auto;
+  .popup-close-btn {
+    order: 1;
+  }
+
+  .popup-toggle {
+    font-size: 2rem;
+    height: 2rem;
+  }
+
+  ul.popup-pr-type {
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .popup-portfolio-img {
+    width: 100%;
     height: auto;
```

Comment:
- [ ] Kindly reduce the `height` of the popup image, it too big compared to the Figma design

### Figma's Design
![Screenshot from 2023-04-19 13-38-59](https://user-images.githubusercontent.com/91142432/233456699-55a9954a-bd83-4d97-93cd-80165c368aa1.png)

### Your Design :point_down: 
![Screenshot from 2023-04-20 18-19-33](https://user-images.githubusercontent.com/91142432/233456796-07fcce6e-2291-4e45-9d3f-26ada6952c3b.png)

Reply:
Thanks